apps/tmbitwarden.sh

```diff
@@ -78,22 +92,126 @@ cd /root
 curl_to_dir "https://raw.githubusercontent.com/bitwarden/core/master/scripts" "bitwarden.sh" "/root"
 chmod +x /root/bitwarden.sh
 check_command ./bitwarden.sh install
-sed -i "s|http_port.*|http_port: 8080|g" /root/bwdata/config.yml
-sed -i "s|https_port.*|https_port: 8443|g" /root/bwdata/config.yml
+sed -i "s|http_port.*|http_port: 5178|g" /root/bwdata/config.yml
+sed -i "s|https_port.*|https_port: 5179|g" /root/bwdata/config.yml
+# Get Subdomain from config.yml and change it to https
+SUBDOMAIN=$(grep ^url /root/bwdata/config.yml)
+SUBDOMAIN=${SUBDOMAIN##*url: http://}
+sed -i "s|^url: .*|url: https://$SUBDOMAIN|g" /root/bwdata/config.yml
+sed -i 's|http://|https://|g' /root/bwdata/env/global.override.env
 check_command ./bitwarden.sh rebuild
 check_command ./bitwarden.sh start
-if check_command ./bitwarden.sh updatedb
+check_command ./bitwarden.sh updatedb
+
+# Produce reverse-proxy config and get lets-encrypt certificate
+msg_box "We will not setup the Apache Proxy that will act as TLS front for your Bitwarden installation."
+
+# Curl the lib another time to get the correct HTTPS_CONF
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
```

Comment:
I remember that you've said, that there is an alternative to this notation. What was it? I currently don't remember.
Travis doesn't seem to like this notation...

Reply:
was it `source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)` ?

Reply:
Maybe, but let's leave it as is. :)